browser/permissions/permission_manager_browsertest.cc

```diff
@@ -120,6 +123,95 @@ class PermissionManagerBrowserTest : public InProcessBrowserTest {
     return context->IsPendingGroupedRequestsEmptyForTesting();
   }
 
+  void TestRequestPermissions(Profile* profile1, Profile* profile2) {
```

Comment:
It seems like there are many copied codes from RequestPermissions test case here which seems unnecessary to me.
It would be more clear to just have a simple ethereum permission added and check if it only appears in one of the profile via checking host_content_settings_map.